### PR TITLE
content(blog): migrate missing blog post

### PR DESCRIPTION
**Pull request by jdrueckert**
https://terasology.org/2021/09/11/teraspotlight-gsoc-2021-summary.html

### CODE REVIEW

File: blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    <h5 class="card-text">Quality Improvements for Light & Shadow</h5>
+    <hr />
+    <div>
+   <a class="text-warning" href="https://summerofcode.withgoogle.com/organizations/5338575677161472/#4719812875386880">PROJECT CARD</a>
+    </div>
+   </div>
+   </div>
+</div>
+
+  Alluri (aka _@ahv15_) overhauled the [`LightAndShadow`](https://github.com/Terasology/lightandshadow) module including removal of unused code and refactorings towards proper event-based Entity-Component-System architecture.
```
Comment (skaldarnar) [inline blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md]:
```suggestion
  Alluri (aka _@ahv15_) overhauled the [LightAndShadow](https://github.com/Terasology/lightandshadow) module including removal of unused code and refactorings towards proper event-based Entity-Component-System architecture.
```

File: blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  Further, Alluri fixed a lot of gameplay bugs related to world generation, starting and respawning inventory, weapon drops and pickups, et cetera.
+  In terms of features, the student introduced a pre-game phase designed to balance delays in team setup both with regards to players joining and teams discussing strategy.
+  This pre-game phase also gives players time to buy in the new shop screen that was extracted from our tower-defense gameplay `GooeyDefence`.
```
Comment (skaldarnar) [inline blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md]:
```suggestion
  This pre-game phase also gives players time to buy in the new shop screen that was extracted from our tower-defense gameplay [GooeyDefence](https://github.com/Terasology/GooeyDefence).
```

File: blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  Accessible throughout all game stages is the new statistics screen that shows the player distribution across the teams as well as stats on their deaths, kills and scored flag points.
+  The final feature, active defensive structures to be placed by players to affect players of the opposing team is still pending in the final review cycles.
+
+  <div align="center">
+    <img src="./las.jpg" width="60%" />
+    <figcaption>Light & Shadow - Shop Screen, Pre-game Phase with separating Domes and Team Balancing, Statistics Screen</figcaption>
+  </div>
```
Comment (skaldarnar) [inline blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md:48]: The caption is rendered quite small, maybe the CSS is not picked up properly? 

image

File: blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+   <div class="col-9"> 
+     <p class="card-title text-muted">Yash Patel</p>
+    <h5 class="card-text">Migrate Web Presence to new Module Site</h5>
+    <hr />
+    <div>
+   <a class="text-warning" href="https://summerofcode.withgoogle.com/organizations/5338575677161472/#5855615500419072">PROJECT CARD</a>
+    </div>
+   </div>
+   </div>
+</div>
```
Comment (skaldarnar) [inline blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md:142]: This looks a bit off, not sure why.

image


this is how it looks on the old page

image

Comment (jdrueckert) [reply blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md:142]: the module site replacement of these cards is not 100% equal. for instance, clicking on the profile picture will not bring you to the contributor's github profile as it does on the current home page.
we need to fix this, but I propose we do that in a separate follow-up PR

**Review by skaldarnar — DISMISSED** [review]
Let's ignore the formatting issues, as they are the same as on previous TeraSpotlight post, as you pointed out.

File: blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md (hunk 3 above)
Comment (jdrueckert) [reply blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md:48]: should be fixed now

File: blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md (hunk 4 above)
Comment (skaldarnar) [reply blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md:142]: I think Gatsby works well with [MDX](https://mdxjs.com/), some markdown extension "for the component area" - basically a way to use custom components in markdown. See [Adding MDX Pages](https://www.gatsbyjs.com/docs/how-to/routing/mdx/) from the Gatsby documentation, for instance.

**Review by skaldarnar — APPROVED** [review]

**Pull request merged**